.golangci.yml

```diff
@@ -143,13 +143,6 @@ linters-settings:
         alias: expv1alpha4
       - pkg: sigs.k8s.io/cluster-api/exp/api/v1beta1
         alias: expv1
-      # CAPI exp addons
-      - pkg: sigs.k8s.io/cluster-api/internal/apis/core/exp/addons/v1alpha3
-        alias: addonsv1alpha3
-      - pkg: sigs.k8s.io/cluster-api/internal/apis/core/exp/addons/v1alpha4
-        alias: addonsv1alpha4
-      - pkg: sigs.k8s.io/cluster-api/exp/addons/api/v1beta1
-        alias: addonsv1
```

Comment:
Following from https://github.com/kubernetes-sigs/cluster-api/pull/11943/files#r1986999982 we should keep these aliases just with the new paths